<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgPatternOptions</class>
 <widget class="QWidget" name="WdgPatternOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>481</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabPatterns">
      <attribute name="title">
       <string>Pattern</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lblPattern">
         <property name="text">
          <string>&amp;Pattern:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>patternChooser</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KisPatternChooser" name="patternChooser" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTransform">
      <attribute name="title">
       <string>Transform</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>10</number>
         </property>
         <property name="verticalSpacing">
          <number>10</number>
         </property>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,0,1">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="KisDoubleSliderSpinBox" name="spbScaleWidth">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="prefix">
              <string>X: </string>
             </property>
             <property name="suffix">
              <string>%</string>
             </property>
             <property name="value">
              <double>99.989999999999995</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KoAspectButton" name="btnLockAspectRatio" native="true"/>
           </item>
           <item>
            <widget class="KisDoubleSliderSpinBox" name="spbScaleHeight">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="prefix">
              <string>Y: </string>
             </property>
             <property name="suffix">
              <string>%</string>
             </property>
             <property name="value">
              <double>99.989999999999995</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="KisSliderSpinBox" name="spbOffsetX">
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="prefix">
              <string>X: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KisSliderSpinBox" name="spbOffsetY">
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="prefix">
              <string>Y: </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1">
           <item>
            <widget class="KisDoubleSliderSpinBox" name="sldShearX">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="prefix">
              <string>X: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KisDoubleSliderSpinBox" name="sldShearY">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="prefix">
              <string>Y: </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="KisAngleSelector" name="angleSelectorRotationZ" native="true">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="container3DRotation" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="spacing">
                 <number>10</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="KisAngleSelector" name="angleSelectorRotationX" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="KisAngleSelector" name="angleSelectorRotationY" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Size:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Position:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Shear:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Rotation</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Align to Pixel Grid:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QCheckBox" name="checkBoxAlignToPixelGrid"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="KisSliderSpinBox" name="sliderAlignToPixelGridX"/>
             </item>
             <item>
              <widget class="KisSliderSpinBox" name="sliderAlignToPixelGridY"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>113</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisSliderSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KoAspectButton</class>
   <extends>QWidget</extends>
   <header>KoAspectButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisAngleSelector</class>
   <extends>QWidget</extends>
   <header location="global">KisAngleSelector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisPatternChooser</class>
   <extends>QWidget</extends>
   <header location="global">kis_pattern_chooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
